packages/module/src/ConsoleInternal/components/markdown-view.tsx

```diff
@@ -234,7 +234,7 @@ const IFrameMarkdownView: React.FC<InnerSyncMarkdownProps> = ({
   body {
     background-color: transparent !important;
     color: ${isEmpty ? '#999' : '#333'};
-    font-family: var(--pf-v5-global--FontFamily--sans-serif);
+    font-family: var(--pf-v6-global--FontFamily--sans-serif);
```

Comment:
If this page has patternfly on it, you should get this for free, but the body font-family token is `--pf-t--global--font--family--body`